Libplanet.Tests/Store/FileStoreFixture.cs

```diff
@@ -18,9 +20,18 @@ public FileStoreFixture()
 
         public void Dispose()
         {
-            if (Directory.Exists(Path))
+            while (Directory.Exists(Path))
             {
-                Directory.Delete(Path, true);
+                try
+                {
+                    Directory.Delete(Path, true);
+                }
+                catch (IOException)
+                {
+                    Log.Debug("IOException occurred during dispose()");
+                }
+
+                Thread.Sleep(0);
```

Comment:
Is this line necessary?

Reply:
This line causes context switching, which is needed to prevent exceptions.